.gitignore

```diff
@@ -110,3 +110,4 @@ client
 __debug_bin
 report.xml
 go.*.orig
+config.mk
```

Comment:
nit: `config.mk` "sounds" like it should be part of the repository. Should we call this `override.mk` or `custom.mk`?

Reply:
I'm ok changing that into something else, but `override.mk` or `custom.mk` i think is not so obvious. maybe have `config.mk` instead of `default-config.mk` and use `config.override.mk` to override the default config.